sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/webflux/ReactorUtils.java

```diff
@@ -1,33 +1,36 @@
 package io.sentry.spring.jakarta.webflux;
 
-import io.sentry.IHub;
+import io.sentry.IScopes;
 import io.sentry.Sentry;
 import org.jetbrains.annotations.ApiStatus;
 import org.jetbrains.annotations.NotNull;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 import reactor.util.context.Context;
 
+// TODO deprecate and replace with "withSentryScopes" etc.
 @ApiStatus.Experimental
 public final class ReactorUtils {
 
   /**
-   * Writes the current Sentry {@link IHub} to the {@link Context} and uses {@link
+   * Writes the current Sentry {@link IScopes} to the {@link Context} and uses {@link
    * io.micrometer.context.ThreadLocalAccessor} to propagate it.
    *
    * <p>This requires - reactor.core.publisher.Hooks#enableAutomaticContextPropagation() to be
    * enabled - having `io.micrometer:context-propagation:1.0.2+` (provided by Spring Boot 3.0.3+) -
    * having `io.projectreactor:reactor-core:3.5.3+` (provided by Spring Boot 3.0.3+)
    */
   @ApiStatus.Experimental
+  @SuppressWarnings("deprecation")
   public static <T> Mono<T> withSentry(final @NotNull Mono<T> mono) {
-    final @NotNull IHub oldHub = Sentry.getCurrentHub();
-    final @NotNull IHub clonedHub = oldHub.clone();
+    final @NotNull IScopes oldHub = Sentry.getCurrentScopes();
```

Comment:
Rename to `oldScopes` and `clonedScopes`/`forkedScopes`?

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366